NoRedInk/elm-blogger
# Move things to be like API-components

So, this includes a few things:
- A top-level thing called `Creator`, which is our entry point now for the UI involving editing markdown
- The Editor component has been ported to use the elm-api-component model and moved to `Component.Editor`
- Include babel to resolve some errors
- `Main` files for each component should be what the old `API` files did. `API` files now include default things that you need for setup, as well as the mailboxes etc
## QA steps:
- Make sure editor works on homepage still
- Make sure that default text shows up
